src/Workleap.OpenApi.MSBuild/OasdiffManager.cs

```diff
@@ -59,9 +59,19 @@ public async Task RunOasdiffAsync(IReadOnlyCollection<string> openApiSpecFiles,
             this._loggerWrapper.LogMessage("- Specification file path: {0}", MessageImportance.High, baseSpecFile);
             this._loggerWrapper.LogMessage("- Specification generated from code path: {0} \n", MessageImportance.High, generatedSpecFilePath);
             
-            var result = await this._processWrapper.RunProcessAsync(oasdiffExecutePath, new[] { "diff", baseSpecFile, generatedSpecFilePath, "--exclude-elements", "description,examples,title,summary", "-f", "text", "-o" }, cancellationToken);
-            this._loggerWrapper.LogMessage(result.StandardOutput, MessageImportance.High);
-            if (!string.IsNullOrEmpty(result.StandardError))
+            var result = await this._processWrapper.RunProcessAsync(oasdiffExecutePath, new[] { "diff", baseSpecFile, generatedSpecFilePath, "--exclude-elements", "description,examples,title,summary", "-o" }, cancellationToken);
+            if (string.IsNullOrEmpty(result.StandardError))
+            {
+                var isChangesDetected = result.ExitCode != 0;
+                this._loggerWrapper.LogMessage("Exit code {0}", MessageImportance.Normal, result.ExitCode);
+                if (isChangesDetected)
+                {
+                    this._loggerWrapper.LogWarning($"Server doesn't match the OpenAPI file: {fileName}. Check the following logs for more details.");
```

Comment:
> Server doesn't match the OpenAPI file

Like in the sense of "Your web API does not implement the following OpenAPI specification: {fileName}"?